bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -472,17 +478,20 @@ private void retrieveBackgroundColor() {
 		textBarForRetrievingTheRightColor.dispose();
 	}
 
+
 	private void createSearchTools() {
 		searchTools = new AccessibleToolBar(searchContainer);
 		GridDataFactory.fillDefaults().grab(false, true).align(GridData.END, GridData.END).applyTo(searchTools);
 
+		@SuppressWarnings("unused")
+		ToolItem separator = searchTools.createToolItem(SWT.SEPARATOR);
```

Comment:
Why store the item in a variable if it unused anyway? I think you can simply remove the variable along with the warning suppression.

Reply:
I agree!

Reply:
Removed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/c7042b7e8f9f3aec27f4d038fa10befb832d8214